Sources/SourceKit/SourceKitServer.swift

```diff
@@ -259,6 +285,10 @@ extension SourceKitServer {
       )
     }
 
+    if let workspace = self.workspace {
+      workspace.buildSettings.delegate = self
```

Comment:
We just set a workspace, so `self.workspace!.buildSettings.delegate = self`?

Reply:
I prefer to keep the if let just for safety

Reply:
I'd rather `assert(self.workspace != nil)`.  Having a workspace is the whole point of the last `if` block above this code.

Reply:
Done